Linear_Regression/LinearRegression Exercises.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
<p>This is the same model as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Thanks fixed</p><p> </p>